docs/source/install.rst

```diff
@@ -119,15 +119,23 @@ Compilation and installation from source code
 The classical way of installing PROJ is via the source code distribution. The
 most recent version is available from the :ref:`download page<current_release>`.
 
-PROJ requires C and C++11 compilers. It also requires SQLite3 (headers,
-library and executable).
-
 You will need that and at least the standard *proj-datumgrid* package for a
 successful installation.
 
 The following guides show how to compile and install the software using the
 Autotools and CMake build systems.
 
+Build requirements
+--------------------------------------------------------------------------------
+
+PROJ 6.2 or later
```

Comment:
I suggest leaving this out and just have this section state what is necessary for whatever is the current version. It is made clear in the docs which PROJ version they refer to

Reply:
done